### ISSUE TITLE
Ad blockers

### ISSUE THREAD

**Issue by yaronvhop** [elastic/apm-agent-rum-js#1078]
it seems that APM RUM is blocked by `ublock origin` ad blocker.

The rule:

https://github.com/easylist/easylist/commit/e0840e06809c594aa999edfab1de8f428241fb05

Any way to get pass it? Might be a option a to have that we can change the URL to avoid those ad blockers blocking performance data?

Thanks

**Comment by gadirotenberg** [comment]
It seems that the `rum/events` suffix of the URL is inlined in the code:
https://github.com/elastic/apm-agent-rum-js/blob/7ae7e395fc5cb45252b1fa92c6ef8675e98e0249/packages/rum-core/src/common/apm-server.js#L339

It will be great to have it as a parameter. On the server side, a possible usage for users that deploy APM using k8s, can be setting an ingress rule to map a suffix of choice to `rum/events`.

**Comment by alexbode** [comment]
I am seeing the same problem of being blocked in the firefox and brave browser. Anyway to add for example `serverEndpoint` to the Config and have the  
```
const endPoint = cfg.get('serverUrl') + cfg.get('serverEndpoint')
```
like @gadirotenberg mentioned?
Or how can someone else make that changes?

**Comment by vigneshshanmugam** [comment]
Anyone wants to do a PR to allow customizing the `serverUrlPrefix` option? It should be fairly small change.

**Comment by alexbode** [comment]
I can make the PR

**Comment by vigneshshanmugam** [comment]
fixed by #1081
